target-platform/target-platform.target

```diff
@@ -66,7 +66,7 @@
 				<dependency>
 					<groupId>ch.qos.logback</groupId>
 					<artifactId>logback-classic</artifactId>
-					<version>1.2.11</version>
+					<version>1.4.6</version>
```

Comment:
```suggestion
					<version>1.2.12</version>
```
Since logback 1.3 slf4j-2 is required, but we cannot use that yet in the Eclipse platform since mina-sshd is not compatible.
I have implemented a fix in https://github.com/apache/mina-sshd/pull/336 but it has not been released.

See also https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/981.

I'll update to logback 1.4 as soon as the new mina-sshd release is available, but until that we have to stay with 1.2 :/
The compiler error is then easy to fix :)